manifests/cluster-install/rbac/argo-events-cluster-role.yaml

```diff
@@ -3,6 +3,20 @@ kind: ClusterRole
 metadata:
   name: argo-events-role
 rules:
+  - apiGroups:
```

Comment:
This rules need to be refined. For example, `configmaps` has been defined in line 46.

Reply:
Did not see `configmap` 🤦  - Will update.